pyuvdata/uvdata/uvh5.py

```diff
@@ -795,7 +875,6 @@ def _read_header_with_fast_meta(
             "ant_1_array",
             "ant_2_array",
             "phase_center_id_array",
-            "Nbls",
```

Comment:
Why did you take `Nbls` out of this list?

Reply:
It was actually in the list twice.